CMSgov/onemac
# OY2-11585: Waiver Package Details View: Base Waivers and Waiver Renewals

Story: https://qmacbis.atlassian.net/browse/OY2-11585
Endpoint: https://d2tqfrkhmyjqi3.cloudfront.net/

### Details
Finalizing the contents and structure of the Waiver Package Details page for Base Waivers and Renewals

### Changes
- added Effective Date and Proposed Effective Date
- added Product Team approved text for when there are no Package Actions available
- created the seed data for the different statuses for the Waivers.

### Test Plan
1. Login as a user who can view Waiver Packages
2. Goto the Package Dashboard, Waivers tab
3. click on the Waiver Number for a Base Waiver in "Package In Review" status
  a. Verify "Package In Review" is at top of card
  b. Verify that Withdraw Package is available
  c. Verify that 90th Day Clock has a value
  d. Verify other Waiver details are correct (Waiver Authority, Waiver Number, Type, State, Date Submitted Proposed Effective Date (should be N/A)
4. Look at the Waiver details for "Approved" package
5. Verify that the Effective Date is shown
6. Verify that package actions is the text "No actions are currently available for this submission."
7. Look at the waiver details for a package in "Terminated" status
8. Verify no clocks are shown
